front/public/locales/fr/rollingstock.json

```diff
@@ -71,7 +71,8 @@
     "rollingStockDuplicateName": "Un matériel roulant du même nom existe déjà.",
     "rollingStockNotAdded": "Le matériel roulant n'a pas été ajouté.",
     "rollingStockNotUpdated": "Le matériel roulant n'a pas été mis à jour.",
-    "rollingStockNotDeleted": "Le matériel roulant n'a pas été supprimé."
+    "rollingStockNotDeleted": "Le matériel roulant n'a pas été supprimé.",
+    "invalidEffortCurves": "La vitesse et l'effort doivent avoir au moins deux valeurs (les lignes ayant au moins un champ vide sont ignorées) , et les valeurs de vitesse doivent être uniques (sans doublons)."
```

Comment:
I think it would be nicer for the user if he had the info of which curves are invalid in the message.
Something like : 
```
Les courbes suivantes sont invalides : Standard > 1000V > 25000V > M4US. Chaque courbe doit posséder au moins 2 combinaisons de valeurs vitesse / effort.
```

Reply:
Done